src/editor/deserialize.js

```diff
@@ -99,7 +108,9 @@ function parseElement(n, partCreator, lastNode, state) {
         case "LI": {
             const indent = "  ".repeat(state.listDepth - 1);
             if (n.parentElement.nodeName === "OL") {
-                return partCreator.plain(`${indent}1. `);
+                // The markdown parser doesn't do nested indexed lists at all, but this supports it anyway.
+                let index = state.listIndex[state.listIndex.length - 1]++;
```

Comment:
I'd prefer to write this out a bit more, something like:
```suggestion
                let index = state.listIndex[state.listIndex.length - 1];
                state.listIndex[state.listIndex.length - 1] += 1;
```